Documentation/cmdref/cilium_service_update.md

```diff
@@ -11,6 +11,7 @@ cilium service update [flags]
 ### Options
 
 ```
+      --backend-weights uints       Backend weights (0 means maintenance) (default [])
```

Comment:
Could you mention that for now it either 0 or 1.

Reply:
yeah, will do that, thanks